build_catalog.sh

```diff
@@ -126,10 +124,19 @@ $CONTAINER_ENGINE tag "$BUNDLE_IMAGE:$CURRENT_COMMIT" "$BUNDLE_IMAGE:latest"
 log "Pushing $BUNDLE_IMAGE:$CURRENT_COMMIT"
 $engine_cmd push "$BUNDLE_IMAGE:$CURRENT_COMMIT"
 
-if [[ -z "${OPM_EXECUTABLE:-}" ]]; then
+# We need an up to date version of opm executable
+opm_local_executable=$(which opm)
```

Comment:
This will cause the script to exit early due to `-e` if `which` fails. You can cheat with something like:
```suggestion
opm_local_executable=$(which opm || true)
```

Reply:
d'oh... I didn't mean to push that yet... Fixed in a48a0bf